ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/BaseLoadBalancer.java

```diff
@@ -182,8 +189,7 @@ void initWithConfig(IClientConfig clientConfig, IRule rule, IPing ping) {
         if (ping instanceof AbstractLoadBalancerPing) {
             ((AbstractLoadBalancerPing) ping).setLoadBalancer(this);
         }
-        logger.info("Client:" + name + " instantiated a LoadBalancer:"
-                + toString());
+        logger.info("Client: {} instantiated a LoadBalancer: {}", name, toString());

```

Comment:
should you pass this instead of calling toString()?  That way it won't evaluate toString until it knows it should be logging?